GetandTake/Configuration/CustomExceptionHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Diagnostics;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace GetandTake.Configuration;
+
+public static class CustomExceptionHandler
+{
+    public static void ExceptionHandler(this WebApplication app)
+    {
+        app.UseExceptionHandler(exceptionHandlerApp =>
+        {
+            exceptionHandlerApp.Run(async context =>
+            {
+                context.Response.StatusCode = StatusCodes.Status500InternalServerError;
+                context.Response.ContentType = Text.Plain;
+
+                await context.Response.WriteAsync("An exception was thrown.");
+
+                var exceptionHandlerPathFeature =
+                    context.Features.Get<IExceptionHandlerPathFeature>();
+                if (exceptionHandlerPathFeature == null)
```

Comment:
General advise: **always** add empty rows between your loops/conditions to avoid them overlapping with other code parts, it is a good practice